support/ref-docs/src/index.ts

```diff
@@ -0,0 +1,67 @@
+import { readdir as fsReadDir, writeFile as fsWriteFile } from 'fs';
+import { join as joinPaths } from 'path';
+import { promisify } from 'util';
+import { ApiModel } from '@microsoft/api-extractor-model';
+import remarkStringify = require('remark-stringify');
+import remarkToc = require('remark-toc');
+import unified = require('unified');
+import { VFile } from 'vfile';
+import documenter from './unist-attacher';
+
+/** Directory that contains API models. */
+const API_MODELS_PATH = joinPaths(__dirname, '..', 'models');
```

Comment:
one day... maybe we take paths like this and `OUTPUT_PATH` below as CLI parameters. for today, this is great!